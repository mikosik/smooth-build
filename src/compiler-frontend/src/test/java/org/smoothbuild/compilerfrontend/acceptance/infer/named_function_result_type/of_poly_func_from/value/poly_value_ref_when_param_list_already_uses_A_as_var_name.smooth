[A] emptyArray = [];
myFunc(A a) = emptyArray;