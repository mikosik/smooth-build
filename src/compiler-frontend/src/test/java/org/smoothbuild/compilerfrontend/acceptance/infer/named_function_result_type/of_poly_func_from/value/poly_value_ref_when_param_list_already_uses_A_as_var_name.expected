[A] emptyArray<A>
  = [];
[B] myFunc<A,B>(A a)
  = emptyArray<B>;
